install/install.go

```diff
@@ -153,6 +161,71 @@ func (i *Installer) InstallArgoCDServer() {
 	argoCDServerControllerDeployment.Spec.Template.Spec.InitContainers[0].ImagePullPolicy = apiv1.PullPolicy(i.ImagePullPolicy)
 	argoCDServerControllerDeployment.Spec.Template.Spec.Containers[0].Image = i.ServerImage
 	argoCDServerControllerDeployment.Spec.Template.Spec.Containers[0].ImagePullPolicy = apiv1.PullPolicy(i.ImagePullPolicy)
+	if i.InstallOptions.ConfigSuperuser {
+		inputReader := bufio.NewReader(os.Stdin)
+
+		fmt.Print("*** Please enter the name of a Kubernetes secret to store the user data: ")
+		secretName, err := inputReader.ReadString('\n')
+		if err != nil {
+			log.Fatal(err)
+		}
+		secretName = strings.Trim(secretName, "\n")
```

Comment:
Secret name is specified in config map, so user might specify config map and provide different secret name here.  Please use default secret name here or secret name from provided config map instead of asking user to retype it.